cmd/query.go

```diff
@@ -254,7 +254,7 @@ $ %s q bal ibc-1 --ibc-denoms`, appName, appName, appName)),
 						switch {
 						case c.Amount.Equal(sdk.NewInt(0)):
 						case c.Denom == d.IBCDenom():
-							out = append(out, sdk.NewCoin(d.GetFullDenomPath(), c.Amount))
+							out = append(out, sdk.Coin{ Denom: d.IBCDenom(), Amount: c.Amount })
```

Comment:
any reason for not using `NewCoin`? `NewCoin` does validation logic so usually it is better practice to use it since it provides stronger guarantees, but it doesn't make much of a difference. I guess in this case, `IBCDeom()` is probably doing some of that validation as well

Reply:
NewCoin was creating the error based on the validation check...

Reply:
Hmm.  I thought the purpose was to output the full path, not the IBC/<hash> denom (in order to match the other output the relayer puts out).  If not, then that is correct..

Reply:
Updated to use NewCoin() with d.IBCDenom() call

Reply:
Based on my interactions with the cli, the relayer doesn't not smartly handle full path denoms provided as inputs. I think for now it is probably best to output `ibc/hash` since it can then be copy/pasted as an argument to `rly tx transfer` cmd (this is being done by your changes). There could be a flag to opt to receive back the full path. 

Coin validation doesn't matter at all here really since this is just the result of a query